This file is part of MXE.
See index.html for further information.

From 8b8c6c188cca7c4ae995c803cffe82bef73c91a4 Mon Sep 17 00:00:00 2001
From: Boris Nagaev <bnagaev@gmail.com>
Date: Mon, 7 Sep 2015 23:01:35 +0100
Subject: [PATCH 4/5] define missing types

We do not use types CERT_NAME_BLOB and CRYPT_HASH_BLOB,
though they are used in mprapi.h. And are not defined!
---
 mysys/my_gethwaddr.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/mysys/my_gethwaddr.c b/mysys/my_gethwaddr.c
index 780c765..e2cd321 100644
--- a/mysys/my_gethwaddr.c
+++ b/mysys/my_gethwaddr.c
@@ -137,6 +137,9 @@ my_bool my_gethwaddr(uchar *to)
 #define VOID void
 #endif
 
+// mprapi.h uses these undefined types
+#define CERT_NAME_BLOB int
+#define CRYPT_HASH_BLOB int
 #include <iphlpapi.h>
 
 /*
-- 
2.1.4

